Makefile

```diff
@@ -52,6 +52,7 @@ generate:
 webdev:
 	make -C frontends/web dev
 weblint:
+	yarn --cwd=${WEBROOT} install
```

Comment:
but you already added this command to scripts/ci.sh before running "make weblint", plus it is also run in "make generate", which you need to execute before any webdev, right.

maybe don't add yarn install here at all?

Reply:
Could `envinit` be the place to do `yarn install`?

Reply:
No, `envinit` installs stuff to the host system (not good, should be improved some day). `yarn install` is local to the repo.

Reply:
@x1ddos good catch. Was a leftover :scream: